src/Api.js

```diff
@@ -21,4 +21,18 @@ export function GetComments(review_id) {
     return ncgamesAPI.get(`reviews/${review_id}/comments`).then((res) => {
         return res.data
     }).catch((err) => { console.log(err, "<- GetComments") })
+}
+
+export function GetVotes(review_id) {
+    return ncgamesAPI.patch(`/reviews/${review_id}`, {inc_votes: 1})
+    .then((res) => {
+    return res.data
+    }).catch((err) => {console.log(err, "<- GetVotes") })
+}
+
+export function RemoveVotes(review_id) {
+    return ncgamesAPI.patch(`/reviews/${review_id}`, {inc_votes: -1})
+    .then((res) => {
+    return res.data
+    }).catch((err) => {console.log(err, "<- GetVotes") })
```

Comment:
These two patch requests are almost identical, can you think of a way to refactor these two requests into one request?